etc/generate-envoy-config/envoy.libsonnet

```diff
@@ -188,7 +285,8 @@
       tcp_keepalive: {},
     },
   },
-  GRPCCluster: self.defaultCluster {
+
+  defaultGRPCCluster:: self.defaultCluster {
```

Comment:
why is this config necessary for grpc?

Reply:
Needs an empty http2_protocol_options to tell envoy that the upstream protocol is http2.  This called "h2c" (http2 over cleartext).  Normally HTTP/2 is negotiated during ALPN, which is part of TLS.

gRPC mandates HTTP/2, but Envoy doesn't know that this cluster will be used for gRPC.  If pachd were doing TLS, then we'd autonegotiate it, but since it's not, we have to force http2 here.

Fun story... one time I randomly ran into someone who was working on Envoy and the empty `http2_protocol_options` was the first thing they mentioned 😂